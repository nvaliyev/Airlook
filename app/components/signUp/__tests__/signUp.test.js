import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import SignUp from '..';
describe(' SignUp', () => {
    test(' SignUp snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(< SignUp/>);
        expect(snapshot).toMatchSnapshot();
    });
});


