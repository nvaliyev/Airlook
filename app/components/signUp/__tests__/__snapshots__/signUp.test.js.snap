// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` SignUp  SignUp snapshot 1`] = `
<View
  style={
    Object {
      "alignContent": "center",
      "alignItems": "center",
      "backgroundColor": "#f7f9fc",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
>
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    placeholder="Username:"
    rejectResponderTermination={true}
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#F7F9FC",
        "borderColor": "#5d6fc4",
        "borderRadius": 5,
        "color": "#6f347c",
        "elevation": 10,
        "height": 65,
        "marginBottom": 6,
        "marginHorizontal": 25,
        "paddingHorizontal": 25,
        "shadowColor": "black",
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    placeholder="Password:"
    rejectResponderTermination={true}
    secureTextEntry={true}
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#F7F9FC",
        "borderColor": "#5d6fc4",
        "borderRadius": 5,
        "color": "#6f347c",
        "elevation": 10,
        "height": 65,
        "marginBottom": 6,
        "marginHorizontal": 25,
        "paddingHorizontal": 25,
        "shadowColor": "black",
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <TouchableOpacity
    activeOpacity={0.2}
  >
    <Icon
      allowFontScaling={false}
      color="#F7F9FC"
      name="eye"
      size={15}
    />
  </TouchableOpacity>
  <ClickButton
    backgroundColor="#6f347c"
    onPress={[Function]}
    text="SignUp"
  />
</View>
`;
